---
title: "BRD Bacterial Affects on the Cattle Industry"
author: "Katelyn B. Kukar, Naupaka Zimmerman"
date: "November 22, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---



# Introduction
The cattle industry is one of the world’s most important agricultural enterprises. Cattle breeding and production serves multiple purposes globally for traction, meat, and milk products to be consumed by the human populations within regions of interest. The largest percentage of cattle enterprises come from both North and South America, due to the origin of domestication and societal usages. In North America specifically, society developed cattle drives to bring beef to urban regions and serve the developing western world. In today’s society, these products are still extensively used to meet not only agricultural demands for beef or dairy, but also byproduct consumables for home, health, and industry. 

The extensive need for cattle within Northern America is thereby important to economic prosperity, making the wellness of the livestock imperative. Bovine respiratory disease is currently the most devastating disease of the United States cattle population with “annual observed incidences ranging from 5 to 44%” of feedlot calves affected [@snowder2006bovine]. From studies done on these incidences it has been determined that grain treatments and stress levels of feed calves are directly related to the infection of livestock with BRD, meaning economic prosperity from cattle can negatively affect the livestock itself [@snowder2006bovine].

In order to determine the economic effects of BRD, Garcia et al. looked at carcass traits to determine how disease changes the quality of meat within affected steers. Those animals treated for BRD have lower quality scores for the meat harvested and a reduction in fat thickness, meaning the diseased cattle had better consumer product worth. Treatment of the animals reduced product quality within the cattle industry [@garcia2010effect].  From this, the industry has concerning evidence to leave affected animals untreated, in order to prevent a loss in meat quality on high valued cattle reservoirs, and now must forecast how the death of young calves may outweigh the price lost in treatment of the entire stock.

Bovine respiratory disease is caused by the growth of pathogenic bacteria in lung tissue due to stress or viral infection. The syndrome itself arises from multiple factors, which are listed in Lekeu et al. as; age, general condition and immune status, environment, changes in food, temperature and humidity, and the presence of infectious agents [@lekeu1996bovine]. Since, the disease has two potential causes (stress or infection) it leads scientists to test the conditions with which a majority of calves incur the symptoms, and the grade of the syndrome that can be treated via vaccination or management practices [@lekeu1996bovine]. BRD is transmitted quickly through cattle stocks via inadequate ventilation from overcrowding and the pathogen alters the animal’s defense mechanism to allow colonization of the lower respiratory tract by bacteria [@fulton2000bovine]. This disease can affect all ages of cattle, but the industry has the highest level of concerns for calves, which have higher rates of mortality from BRD and outbreaks spreading in 60-100% of affected units in tested feedlots, specifically from M. Bovis [@arcangioli2008role].

The cattle industry is incredibly important to the economic prosperity of the United State agricultural industry, with the wellness of young calves becoming an increasingly relevant concern for scientists. With the hopes of studying the bacteria associated in lung and mediastinal lymph node tissues of healthy and BRD-diseased calves, microbiologists hope to divulge the microbiome of the lower respiratory tract in order to conclude which bacteria species may be associated with BRD outbreaks alone [@kelly2017illumina]. 
Here we utilize NGS 16s amplicon analysis to identify the relevant changes in bacterial community composition, within animals sampled, using Illumnia MiSeq from Johnston et al. We hypothesize that substantial bacterial community differences in the microbiome will arise within healthy and diseased samples. We predict rises in stress level will potentially cause lower immunity levels resulting in an increase in abundance and richness for affected cattle, when compared to baseline controls. From the diseased samples, we further assume that the bacterial genera found with the most significant changes in abundance will have potential relationships to pathogenic disease spreading seen in other livestock species or relationships with the environment, immune responses, or dietary instabilities. These found genera may also correlate to additional studies on Bovine Respiratory Disease as supporting evidence for a singular bacterial component of interest.


# Methods

## Sample origin and sequencing

All sample origins and sequencing was done by Johnston et al. and associated authors to obtain the data utilized throughout this analysis.

### Tissue Collection:
38 calves, from a variety of different farms, were selected post-mortem and had cranial lung lobe tissue samples removed, with 32 also having mediastinal lymph node tissues collected. BRD was diagnosed by three independent veterinary laboratories by pathologists through bacterial culture, virological and/or bacteriological PCR and histology for identification [@kelly2017illumina]. 20 samples of each tissue type were simultaneously collected from control healthy calves. These calves were slaughtered at 2.5 months and had clinical health checks done before slaughter [@kelly2017illumina]. One gram of lung tissue was extracted from lesions present on the cranial lobe for each sample type (healthy or affected) with 6 immediately frozen at -80 after collection [@kelly2017illumina]. The remaining samples were placed in RNALater RNA Stabilizing Reagent from Qiagen Inc. and stored based on kit instructions [@kelly2017illumina]. 

### qPCR from Whole Tissue Samples:
DNA extraction of these tissues occurred in a BSLII safety cabinet with Qiagen QIAampe Cador Pathogen Mini Kit after pre-treatments with enzymatic tissue digestion (T2) and for difficult to lyse bacteria (B1) [@kelly2017illumina]. Once treated, a small piece of tissue was homogenized and transferred to a micro centrifuge tube with proteinase K [@kelly2017illumina]. After nucleic acid was extracted RNA was removed using 5ul RNAase A solution for 100ul of purified samples. DNA was purified using Zymo Genomic DNA Clean and Concentrator -10 Kit[@kelly2017illumina]. A Nanodrop spechtrophotometer was used to quantify the DNA [@kelly2017illumina].

### Library Prep:
One hundred and sixteen 16s rRNA gene amplicon libraries were prepared via PCR amplification of an approximate 467 bp region within the 16sRNA gene in bacteria from 50 ng of each of the extracted and purified DNA samples, using molecular water as a control [@kelly2017illumina]. This procedure followed guidelines set up by Illumnia 16s meta-genomic sequencing library protocol, with modifications based on sample type [@kelly2017illumina]. PCR was performed with broad spectrum 16s rRNA primers using Kapa HiFi HotStart 2x ReadyMix DNA polymerase (Kapa Biosystems Ltd., London, UK) [@kelly2017illumina]. Libraries were purified with AMPure XP beads and dual indices and Illumnia sequence adaptors from Illumnia Nextera XT index kits vs B and C were added to target amplicons of interest in a second PCR repeat [@kelly2017illumina]. A Nanodrop spectrophotometer measured for quality and purity before barcoded amplicon libraries were combined in equal concentrations into a single pool. Two ul of negative control were additionally added [@kelly2017illumina]. The library pool was quantified with KAPA SYBR FAST universal qPCR kit with Illumnia Primer Premix and size assessment on an Agilent 2100 Bioanalyser [@kelly2017illumina]. The library was prepped to MiSeq standards and the sequencing run was conducted on the Illumnia MiSeq using the 500 cycle MiSeq reagent kit (v2) with paired 250 bp reads [@kelly2017illumina]. The data was submitted as a BigProject to NCBI under accession number SRP080306 [@kelly2017illumina].


## Computational

All computational analyses were conducted by students at the University of San Francisco and Naupaka Zimmerman on data received from Johnston et al. This sequencing data was evaluated individually for quality and hypotheses were furthered through statistical analyses to determine more information about the microbiome of BRD affected and healthy calves.

The packages required for analysis include: dplyr, tidyr, knitr, ggplot2, dada2, seqinr, mctoolsr, phyloseq, and RcolorBrewer. These can be downloaded through R our through their corresponding web links: https://benjjneb.github.io/dada2/dada-installation.html for dada2 to clean and analyze data, https://github.com/leffj/mctoolsr for mctoolsr.

The BioProject, found under the listed NCBI accession number, needs to have the SRA table initially downloaded using the appropriate Toolkit. A script needs to be run to download the table and also extrapolate all run data from the table as fastaq files with associated quality scores. A fastq dump also is run with a splitting file code to split the forward and reverse reads into forward reads only to be analyzed within a dada pipeline as needed fast files  [@callahan2016].   

The pipeline directly followed is DADA2 with information at https://benjjneb.github.io/dada2/tutorial.html. First the path for the data needs to be established with all fastq file sorted by the indicated run number for all samples. Once sorted graphs of the quality scores can be plotted with the plot quality profile function to determine the cut off point for all sequences. The files are filtered and placed into a filtered region of the folder to use in analysis [@callahan2016]. The sequences are then trimmed following dada pipeline constraints specific to Illumnia MiSeq parameters, which include maximum length required and errors allowed [@callahan2016]. A markdown table is produced to see the runs in and out before and after trimming to indicate the level of processing done on each file type [@callahan2016]. Error models are built to establish a trend of errors within the entirety of the project based on trimming constraints with an expected level of errors to make statistically viable figures; plot errors can visualize these trends [@callahan2016]. Removal of replicated sequences is an important pipeline quality check to ensure no replications skew data later in the pipeline. Dada is then run on all forward reads with MiSeq parameters to look at homopolymers and potential sequencing deficiencies with a post check of the run to evaluate sequences [@callahan2016]. A site by species matrix is produced to look at the sequence length, coupled with a histogram to see trimming parameters. We see all sequences are trimmed to the same length [@callahan2016]. Chimeras are removed to aid in the alignment of the sequences and normalize samples. A table can showcase how many sequences are left at each step of the pipeline to track the level of trimming, alignment, and removal of error using the track function within the dada pipeline [@callahan2016]. Taxonomies are assigned based on documented sequences through the taxa function, and exported as a taxa table for community analysis of known samples. Meta data is merged with taxonomies to bring in all known data about sample origin and sequencing processes from all stages of research on the healthy and affected bovine samples. This information is gathered in a phyloseq object and melted to create a usable data table with known information [@mcmurdie2013]. 

In all resulting statistical analyses the phyloseq object and melted object are applied to use ggplot2 in making tables and figures [@wickham2011ggplot2]. The pruning function removes Not Applicable data in some cases.

# Results

Sequencing Data was compiled into a primary phyloseq object to be utilized within R for comparative analyses [@wickham2011ggplot2]. The data will potentially give rise to information regarding the community aspects of the microbiome within the Bovine Respiratory Epithelium from lung and mediastinal lymph node tissues.

Abundance measures the amount if a population of bacteria within a sample and can show the relative size of a colony residing in a tissue. The more abundant a particular colony the more probable that the colony type is thriving in the environment set forth. Analyzing the amount of bacteria in healthy and affected tissues can lead to information regarding the ability for bacteria to thrive within and calves. 

```{r, add in all libraries, echo=FALSE, message=FALSE}
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")
library("RColorBrewer")
```


```{r, pathway to read in Phyloseq Object, echo=FALSE, message=FALSE}
load("output/phyloseq_obj.RData")
```

```{r, melt Phyloseq with Metadata table, echo=FALSE, message=FALSE}
# Melt Phyloseq
melted_obj <- psmelt(phyloseq_obj)
```

```{r, prune phyloseq, echo=FALSE, message=FALSE}
# apply pruned data set
pruned_phyloseq <- subset_samples(phyloseq_obj)
```

# Analysis of Raw Colonies

**Figure 1:**
```{r, Abundance of Bacteria Found in Sample Types, echo=FALSE}
melted_obj %>%
  ggplot(
       aes(x = BRD_affected_or_healthy_s,
           y = Abundance)) +
  geom_boxplot(alpha = 1) +
  ggtitle("Abundance for Healthy vs. Diseased Calves") +
  xlab("Disease Status") +
  ylab("Abundance") +
  theme_light() +
  scale_y_log10()
```

Figure 1 represents the abundance of bacteria found within the healthy or BRD affected calves post-mortem for both tissue types sampled. From this figure we can see a similar amount of bacteria was present in both types.

Richness is the quantification of different types of species living within a similar niche environment. Looking at the richness within affected and healthy bovine tissue samples will determine if a singular type of bacteria is present or many. If many different bacterial types make up the microbiome many different bacteria will be utilizing the nutritional resources provided by the calf's tissues. 

**Figure 2**:
```{r, Richness of Bacteria, echo=FALSE}
#abundance, boxplot, table of values
plot_richness(pruned_phyloseq,
              x = "BRD_affected_or_healthy_s",
              measures = c("Observed")) +
 xlab("Sample origin") +
 geom_boxplot(width = 0.2) +
 theme_bw() +
 labs(title = "Healthy vs. Affected Richness Metric")
```
Figure 2 represents the richness of bacterial samples found within healthy and affected calves. BRD affected individuals have a larger spread of richness indicating more populations of bacteria within a singular niche.


Shannon Diversity works to combine both the abundance and evenness of a certain community sample set. Shannon Diversity applies a true diversity algorithm that refers to the number of equally abundant types needed for the average proportional abundance of the types to equal that observed in a data set where all types may not actually be equal. It utilizes H to assume the proportion of each species type is indicative of their importance of diversity or the overall biomass of the community. Within bovine tissues the diversity metric will provide insight on how different the microbiomes for healthy and affected calves are. Evenness within the niche can further determine if each species of bacteria represents a similar abundance throughout the tissue or if one species has a higher selection coefficient to compete for resources. In BRD affected calves one species of bacteria may be more prevalent in acquiring nutrition from the tissues it resides in leaving to faster death during bacterial colony expansion. Evenness works as a diversity index to measure the biodiversity in a reference community to see how equal each species variant is numerically.

**Figure 3**:
```{r, Shannon Diversity Metric, echo=FALSE}
# alpha diversity metrics
plot_richness(pruned_phyloseq,
              x = "BRD_affected_or_healthy_s",
              measures = c("Shannon")) +
  xlab("Sample origin") +
  geom_boxplot(width = 0.2) +
  theme_bw() +
  labs(title = "Healthy vs. Affected Diversity Metric")
```
Figure 3 outlines a Shannon Diversity Metric to look at the diversity of the samples from healthy and affected calves. BRD affected samples have a large spread of diversity with a lower mean value than healthy samples.

## Comparison of Healthy and Affected Microbiome

Direct comparisons of the differing microbiomes is important to the juxtaposition of healthy versus affected calves. Distinguishing how the two correlate could lead to evidence in regard to the bacteria responsible for young calf fatalities due to BRD.

Ordination can be used as a exploratory measure for data clustering of similar and different multivariate types. Ordinations plot relationships between community microbiomes based on richness and diversity metrics. The overlay of both population types can showcase outliers in microbiome populations for affected samples and remove common microbiota within both to see the spread of the total community.

**Figure 4**:
```{r, Ordination, echo=FALSE}
# Ordination of Healthy and Affected species distribution
set.seed(28)
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")
# plot ordination without NA
p1 <- phyloseq::plot_ordination(pruned_phyloseq,
                     pruned_ord,
                     type = "samples",
                     color = "BRD_affected_or_healthy_s",
                     title = "Ordination of Healthy and Affected Sequences",
                     label = "Sample_Name_s")
p1
```

Figure 4: The observed Ordination, which distinguishes healthy and BRD affected calves, shows a level of relatedness for the healthy calf microbiome, but a large spread within affected calf microbiome. However, some overlap is denoted showing common microbiota for both sample types.

**Figure 5**:
```{r, Phlyum Comparison of Affected and Non}
# filter the sample name and see the phylum make up

melted_obj %>%
  filter(Sample_Name_s %in% c("31LNBeef", "4LBeef", "2LHF", "19LHF")) %>%
  group_by(Sample_Name_s, Phylum) %>%
  summarise(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = Sample_Name_s,
             y = mean_abund,
             fill = Phylum)) +
    geom_col()
```

Figure 5 represents all Phyla within the samples from healthy calves seen in the most dense cluster of the ordination (Figure 4) and the most spread samples from affected calves (Figure 4). 4LBeef and 31LNBeef represent the affected calves. 4LBeef is the most distinct sample from Lymph Node Tissue with 4L Beef being the most distinct sample from Lung Tissue. Within the Lymph Node only **Phlya** is seen where in Lung Tissue only. In healthy calves there is a mixture of many bacterial types within each tissue sample


## Analysis of Bacterial Types

Grouping the number of bacteria seen within samples with a known bacterial definition can lead to information in regard to the most prevalent Phyla and Genera seen in healthy and affected individuals. These levels of abundance can be used to visualize colony growth and prosperity within calf tissues for asymptomatic (baseline) versus symptomatic calves.

**Table 1**:
```{r, Phylum Table, echo=FALSE}
# Table to look at Phylums
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(BRD_affected_or_healthy_s, Phylum) %>%
  tally() %>%
  spread(key = BRD_affected_or_healthy_s,
         value = n) %>%
  arrange(desc(BRD), desc(healthy)) %>%
  kable()

```

Table 1 looks at the distribution of Phyla throughout the samples tested to see the different types of bacteria represented in the respiratory tract of the calves. 57.14% more bacteria is seen in affected calves for all Phyla observed.


**Table 2:**
```{r, Genus Table, echo=FALSE}
# Genera abundance
melted_obj %>%
  filter(!is.na(Genus)) %>%
  group_by(BRD_affected_or_healthy_s, Genus) %>%
  tally() %>%
  spread(key = BRD_affected_or_healthy_s,
         value = n) %>%
  arrange(desc(BRD), desc(healthy)) %>%
  head(10) %>%
  kable()
```

Table 2 looks at the most relevant genera for affected versus healthy tissue types. 57.14% more bacteria for each type is visualized in affected samples than healthy samples for all tested genera.



# Discussion

Bovine Respiratory Disease is currently the largest threat to the cattle industry due to the high fatality rate of young calves. Death of young calves leads to sever economic losses from to moralities, cost of treatments, reduced feed efficiency, and lower carcass quality as denoted by Garcia et al. These losses create less profit for feedlot owners and cattle breeders who by default cut costs by reducing the quality of the food they feed healthy cows, and the quality of enclosures. Though profits are imperative to the agricultural business these quality cuts increase the spread of BRD by lowering the immunity of cattle via poor nutritional diets and optimizing the conditions for the opportunistic bacteria to invade via co-infection with viruses or commingling of animals from different sources (Holman et al.). Poor ventilation within feedlots heightens the stress of the animals releasing adrenaline into the system to increase cortisol levels, which in turn reduce the immunity within the animal to create adrenal fatigue (Snowder et al.).

Within this paper we evaluate the differences within microbiota found in two lung tissues forms of healthy and known BRD affected calves. We predict that a community analysis will reveal significant differences within the richness and abundance of both sample types to yield evidence to support a decrease in immunity from known, heightened stress conditions. We further predict that the resulting genera deduced from OTU taxonomical analysis will correlate with immune deficiencies, environmental pressures, or dietary instabilities previously documented.
	
Within Figure 1 we decipher the relationship that bacterial abundance has with the samples tested. We deduce that overall bacterial amounts are consistent between both affected and healthy bovine tissue samples. We can conclude that these results indicate that only a certain amount of bacteria can occupy each particular niche at a given time which is consistent with most environmental density curves, since the tissue offers a limited amount of resources to the bacteria meaning competition for these resources is dependent on the selection mechanisms for each bacterial type. If this is true then the community is k-selected and all bacteria forms are utilizing the max amount of resources from each tissue type either affected or healthy. We can hypothesize that the disease-causing bacteria may be more rich than that of the healthy bacteria in affected tissues, causing a decrease in potentially profitable bacterial types that increase the immunity within the cranial lung lobe tissue and mediastinal lymph node tissues collected.
	
To test if the richness is variable between healthy and affected samples Figure 2 looks at the overall spread of each unique species within the tissue samples. This richness plot shows that BRD affected samples have an increased spread of richness, with many outliers compared to healthy control variants. Though the mean richness for healthy calves is higher, the spread for affected calves indicates many species are inhabiting a similar niche. Figure 2 helps support the primary claim that though the total abundance is similar among populations, immune deficiency occurs because the disease-causing bacteria are inhabiting the same niche environment as bacteria in healthy samples causing increased selection pressures that allow the diseased bacteria to out compete the endogenous species. 
	
By, understanding that the affected calves have a greater richness of bacteria within their tissue samples information about immunity can be gathered. Low levels of immunity due to high stress conditions can cause less energy to be put towards creating proteins to fight infection and more energy towards the production of cortisol. If high stress conditions are being create because of cattle transfer and crowding calves may increase the amount of cortisol their body is producing to prevent being trampled by other cows or to maintain a state of alert during long trucking endeavors. A secondary hypothesis to bovine respiratory disease is viral infection which could be caused by overcrowding in feed lots. If cattle were placed in area of low sanitation an increase in bacterial types could accumulate in tissues sampled from manure or co-infection from differing cattle sites. Both of these scenarios would lead to an increase in bacterial richness when compared to healthy calf tissues [@edwards2010control].

Diversity measures the relative sustainability of a certain environment for its inhabitants by looking at richness and evenness. If some species are exponentially taking over the habitat of another evenness is reduced. Figure 3 shows a higher mean diversity level for healthy calves yielding support for a more sustainable environment within healthy tissue types. This environment is more sustainable since there is a balance within the microbiome that resides there, unlike the large spread of data seen for BRD affected samples. The large spread denotes high levels of diversity variability which is indicative of dying bacterial species and exponentially growing species. It is plausible that as BRD symptoms increase so does the relative amounts of harmful bacteria within the respiratory epithelium.
	
Ordination to directly overlay the levels of community analyses for comparing the healthy and affected tissue types sampled, as seen in Figure 4. The clustering of the healthy samples is indicative of similar bacterial types with low levels of diversity among each, while the large spread of affected samples indicates huge amounts of variety within the community. This variation within the community is most likely from decreased immune levels which could be from low quality food reducing the nutritional needs of calves, especially in environments of high density cohabitation or fecal matter. 
	
Though understanding the microbiome of the calf samples at a community level is important to developing hypotheses about the relative state of the immune system, understanding the types of bacteria residing in the tissue can also determine which bacterial forms are exponentially growing and which are decreasing. Table 1 indicates abundance information regarding Phylum to show an overall increase of 54.17% of the bacteria in all Phyla tested. The increase in all Phylum’s could be due to the spread of both beneficial and harmful bacteria during immune responses to growth. This arms race could have led to a reciprocal amount of action from both types of bacteria that remains constant in healthy, control tissues. The same trend is visualized within the Genera analysis, where all affected types increase by more than 50% against healthy samples.
	
The most prevalent Phyla for both healthy and affected tissue is **Firmicutes**. **Firmicutes** is related to the immune system via diet. **Firmicutes** are known gut microbiota that help aid the body in digestion. Many studies have been done on a bacterial link to obesity, and have found the **Firmicutes** help aid in extraction of nutrients from food and stimulate host immunity [@mariat2009firmicutes]. However, when removed from the gut they can also propagate the creation of digestive substances through increased mucus levels [@holman2017nasopharyngeal]. 

The most prevalent Genera for both healthy and affected tissue is **Protovella**. **Protovella** is relevant to this discussion since it is found in the oral cavity and can lead to anerobic infection [@brodsky2010maxillofacial]. The close proximity of feedlot calves transfers bacteria readily. BRD could be associated with low immunity allowing harmful **Protovella** to colonize calf respiratory tracts at a faster rate than those with a normal immune baseline.

When looking at specific affected tissues that showcase the most extreme microbial distinctions, the richness and diversity seen at a community level in all samples is decreased (Figure 5). In comparison to healthy individuals, affected individuals show only one prevalent bacterial community (Figure 5). It can be inferred the the increase in community richness and diversity is due to the different states of BRD, and the unique ways BRD is affecting the individual calf. Much like human immune systems, cattle immunity is dependent on many factors and may yield different results depending on the baseline immunity of the calf, and environmental or dietary factors. Differences in tissue type also must be considered since Mediastinal Lymph Nodes are located on the esophagus that can interact with different bacteria than that of the lungs. **Firmicutes** are gram-positive bacteria known to be important to mucous immunity and are present within the gut, however if these gut microbes are found in abundance near the esophagus it is probable that the excess mucus is causing lung buildup, leading to pneumatic symptoms [@holman2017nasopharyngeal]. In lung tissues **Tenericutes** are present which could be common for calves with BRD because **M. Pneumoniae** is a species within the Phylum **Tenericutes** [@confer2009update]. **M. Pneumoniae** is characterized by the absence of a peptidoglycan cell wall, resulting in resistance to many antibacterial agents and therefore potentially aiding in the symptoms of BRD, and in the increased difficulty farmers are having creating a cure [@confer2009update]. Figure 5 also shows the Phyla in healthy calves from samples closest to the high density region of relatedness seen in Figure 4. These samples have a more diverse scope of Phyla. This is potentially from the affected samples having symptomatic bacteria out-competing normal bacterial levels because of a low immunity induced by stress or co-infection.

	
Overall, the immune system is the body’s defense against infectious organisms and microorganisms that could potentially harm imperative tissues or cells during their colonization. When the immune system recognizes foreign invaders it works to remove them from the environment using a network of cells, tissues, and organs; specifically leukocytes [@jamieson2013role]. However, when overwhelmed with multiple invaders during co-infection the body becomes overwhelmed and will focus reservoirs of energy onto viral infections over bacterial colonization events. In this way BRD can easily infect young calves that are simultaneously fighting off more sever, short-term illnesses. When the viral infection is depleted usually the bacterial one has grown under asymptomatic expression until it can fully infect host tissue [@edwards2010control]. BRD from stress is also plausible since stress induced immunodeficiency is common even in human hosts. Chronic stress is seen when cortisol is circulating in the blood for long period of time to cause digestive issues and high blood pressure. Stress specifically targets the immune system by creating chronic inflammatory conditions and thereby increasing the production of cytokines to allow and autoimmune attack on healthy cells, leaving bacterial hosts able to colonize tissues without leukocyte or phagocytic removal [@reiche2004stress].
	
Both of these cases are plausible given known bovine habitat situations and both can be fixed to increase the economic prosperity of the cattle industry by investing in better transportation and open pasture lifestyles. If the cattle industry pushes for an organic transition to allow grazing with supplemental vitamins the immunity of many calves would remain unaffected by BRD induced by stress or co-infection. 
	
Future directions for this study include identification of specific OTUs and there direct relationship with the immune system seen for bovine calves. More information also needs to be gathered to create quantitative measures for the controlled change in immunity from stress during bovine transportation and in immediate feedlot conditions, compared to bovine calves in open air environments. Dietary and nutritional changes also must be accounted for due to the prevalence of **Protovella** and its relationship with bovine diet over the lifespan. By creating metrics to find the exact bacterium responsible for BRD, and it's associations with immunity, the cattle industry can protect its stock and simultaneously craft better, healthier agricultural methods for bovine and consumers.


# Sources Cited
