---
title: "Final Report: Your Title Here"
author: "Don Francisco"
date: "November 22, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---



# Introduction
The cattle industry is one of the world’s most important agricultural enterprises. Cattle breeding and production serves multiple purposes globally for traction, meat, and milk products to be consumed by the human populations within regions of interest. The largest percentage of cattle enterprises come from both North and South America, due to the origin of domestication and societal usages. In North America specifically, society developed cattle drives to bring beef to urban regions and serve the developing western world. In today’s society, these products are still extensively used to meet not only agricultural demands for beef or dairy, but also byproduct consumables for home, health, and industry. 

The extensive need for cattle within Northern America is thereby important to economic prosperity, making the wellness of the livestock imperative. Bovine respiratory disease is currently the most devastating disease of the United States cattle population with “annual observed incidences ranging from 5 to 44%” of feedlot calves affected (Snowder et al.). From studies done on these incidences it has been determined that grain treatments for feed calves is directly related to the infection of livestock with BRD, meaning economic prosperity from cattle can negatively affect the livestock itself (Snowder et al.).

In order to determine the economic effects of BRD, Garcia et al. looked at carcass traits to determine how disease changes the quality of meat within affected steers. Those animals treated for BRD have lower quality scores for the meat harvested and a reduction in fat thickness, meaning the diseased cattle had better consumer product worth. Treatment of the animals reduced product quality within the cattle industry (Garcia et al.).  From this, the industry has concerning evidence to leave affected animals untreated, in order to prevent a loss in meat quality on high valued cattle reservoirs, and now must forecast how the death of young calves may outweigh the price lost in treatment of the entire stock.

Bovine respiratory disease is caused by the growth of pathogenic bacteria in lung tissue due to stress or viral infection. The syndrome itself arises from multiple factors, which are listed in Lekeu et al. as; age, general condition and immune status, environment, changes in food, temperature and humidity, and the presence of infectious agents (Lekeu et al.). Since, the disease has two potential causes (stress or infection) it leads scientists to test the conditions with which a majority of calves incur the symptoms and the grade of the syndrome that can be treated via vaccination or management practices (Lekeu et al.). BRD is transmitted quickly through cattle stocks via inadequate ventilation from overcrowding and the pathogen alters the animal’s defense mechanism to allow colonization of the lower respiratory tract by bacteria (Fulton et al.). This disease can affect all ages of cattle, but the industry has the highest level of concerns for calves, which have higher rates of mortality from BRD and outbreaks spreading in 60-100% of affected units in tested feedlots, specifically from M. Bovis (Arcangioli et al.).

The cattle industry is incredibly important to the economic prosperity of the United State agricultural industry, with the wellness of young calves has become an increasingly relevant concern for scientists. With the hopes of studying the bacteria associated in lung and mediastinal lymph node tissues of healthy and BRD-diseased calves, microbiologists hope to divulge the microbiome of the lower respiratory tract in order to conclude which bacteria species may be associated with BRD outbreaks alone (Johnston et al.). 
Here we utilize NGS 16s amplicon analysis to identify the relevant bacteria within animals sampled using Illumnia MiSeq from Johnston et al. We hypothesize that substantial bacterial community differences in the microbiome will arise within healthy and diseased samples. From the diseased samples, we further assume that the bacteria found will have potential relationships to pathogenic disease spreading seen in other livestock species or relationships with the environment, immune responses, or dietary instabilities. 
	


# Methods

## Study design

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Sample origin and sequencing

All sample origining and sequencing was done by Johnston et al. and associated authors to obtain the data utilized throughout this analysis.

###Tissue Collection:
38 calves, from a variety of different farms, were selected post-mortem and had cranial lung lobe tissue samples removed, with 32 also having mediastinal lymph node tissues collected. BRD was diagnosed by three independent veterinary laboratories by pathologists through bacterial culture, virological and/or bacteriological PCR and histology for indentification. 20 samples of each tissue type were simultaneously collected from control healthy calves. These calves were slaughtered at 2.5 months and had clinical health checks done before slaughter. One gram of lung tissue was extracted from lesions present on the cranial lobe for each sample type (healthy or affected) with 6 immediately frozen at -80 after collection. The remaining samples were placed in RNALater RNA Stabilizing Reagent from Qiagen Inc. and stored based on kit instructions. 

###qPCR from Whole Tissue Samples:
DNA extraction of these tissues occurred in a BSLII safety cabinet with Qiagen QIAampe Cador Pathogen Mini Kit after pre-treatments with enzymatic tissue digestion (T2) and for difficult to lyse bacteria (B1). Once treated, a small piece of tissue was homogenized and transferred to a microcentrifuge tube with proteinase K. After nucleic acid was extracted RNA was removed using 5ul RNAase A solution for 100ul of purified samples. DNA was purified using Zymo Genomic DNA Clean and Concentrator -10 Kit. A Nanodrop spechtrophotometer was used to quantify the DNA.

###Library Prep:
One hundred and sixteen 16s rRNA gene amplicon libraries were prepared via PCR amplification of an approximate 467 bp region within the 16sRNA gene in bacteria from 50 ng of each of the extracted and purified DNA samples, using molecular water as a control. This procedure followed guidelines set up by Illumnia 16s metagenomic sequencing library protocol, with modifications based on sample type. PCR was performed with broad spectrum 16s rRNA primers using Kapa HiFi HotStart 2x ReadyMix DNA polymerase (Kapa Biosystems Ltd., London, UK). Libraries were purified with AMPure XP beads and dual indices and Illumnia sequence adaptors from Immunia Nextera XT index kits vs B and C were added to target amplicons of interest in a second PCR repeat. A Nanodrop spectrophotometer measured for quality and purity before barcoded amplicon libraries were combined in equal concentrations into a single pool. Two ul of negative control were additionally added. The library pool was quantified with KAPA SYBR FAST universal qPCR kit with Illumnia Primer Premix and size assessment on an Agilent 2100 Bioanalyser. The library was prepped to MiSeq standards and the sequencing run was conducted on the Illumnia MiSeq using the 500 cycle MiSeq reagent kit (v2) with paired 250 bp reads. The data was submitted as a BigProject to NCBI under accession number SRP080306.



## Computational

All computational analyses were conducted by students at the University of San Francisco and Naupaka Zimmerman on data received from Johnston et al. This seqeuncing data was evaluated individually for quality and hypotheses were furthered through statistical analyses to determine more information about the microbiome of BRD affected and healthy calves.

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results


```{r, add in all libraries}
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")
library("RColorBrewer")
```


```{r, pathway to read in Phyloseq Object}
load("output/phyloseq_obj.RData")
```

```{r, melt Phyloseq with Metadata table}
# Melt Phyloseq
melted_obj <- psmelt(phyloseq_obj)
```

```{r}
# apply pruned data set
pruned_phyloseq <- subset_samples(phyloseq_obj)
```


## Subsections are ok in the results section too
#Tables of interest

```{r your-code-here}
# Add code chunks as needed for your analyses
# For most analyses, I would recommend splitting the
# intensive computational part into a seperate R script file
# and then just load your libraries and the data object here
# using the `load()` function. Ask me for clarification if this is
# unclear.
```

**Figure 1:**
```{r, Abundance of Bacteria Found in Sample Types}
melted_obj %>%
  ggplot(
       aes(x = BRD_affected_or_healthy_s,
           y = Abundance)) +
  geom_boxplot(alpha = 1) +
  ggtitle("Abundance for Healthy vs. Diseased Calves") +
  xlab("Disease Status") +
  ylab("Abundance") +
  theme_light() +
  scale_y_log10()
```

Figure 1 represents the abundance of bacteria found within the healthy or BRD affected calves post-mortem for both tissue types sampled. From this figure we can see a similar amount of bacteria was present in both types.

**Figure 2**:
```{r, Richness of Bacteria}
#abundance, boxplot, table of values
plot_richness(pruned_phyloseq,
              x = "BRD_affected_or_healthy_s",
              measures = c("Observed")) +
 xlab("Sample origin") +
 geom_boxplot(width = 0.2) +
 theme_bw() +
 labs(title = "Healthy vs. Affected Richness Metric")
```
Figure 2 represents the richness of bacterial samples found within healthy and affected calves.

**Figure 3**:
```{r, Shannon Diversity Metric}
# alpha diversity metrics
plot_richness(pruned_phyloseq,
              x = "BRD_affected_or_healthy_s",
              measures = c("Shannon")) +
  xlab("Sample origin") +
  geom_boxplot(width = 0.2) +
  theme_bw() +
  labs(title = "Healthy vs. Affected Diversity Metric")
```
Figure 3 outlines a Shannon Diversity Metric to look at the diversity of the samples from healty and affected calves.

**Figure 4**:
```{r, Ordination}
# Ordination of Healthy and Affected species distribution
set.seed(28)
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")
# plot ordination without NA
p1 <- plot_ordination(pruned_phyloseq,
                     pruned_ord,
                     type = "samples",
                     color = "BRD_affected_or_healthy_s",
                     title = "Ordination of Healthy and Affected Sequences")
                     print(p1)
```

Figure 4: The observed Ordination, which distinguishes healthy and BRD affected calves, shows a level of relatedness for the healthy calf microbiome, but a large spread witin affected calf microbiome.

```{r, prune phyloseq for Phylum}
pruned_phyloseq <- subset(pruned_phyloseq, )

```


**Table**:
```{r, Phylum Table}
# Table to look at Phylums 
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  group_by(BRD_affected_or_healthy_s, Phylum) %>%
  tally() %>%
  spread(key = BRD_affected_or_healthy_s,
         value = n) %>%
  arrange(desc(BRD), desc(healthy)) %>%
  kable()

```
Figure 5 looks at the distribution of Phylums throughout the samples tested to see the different types of bacteria represented in the respiratory tract of the calves.


**Table:**
```{r, table to see counts of certain genus}
# want a table to look att he number of times a certain genus occurs in both healthy and affected individuals
melted_obj %>%
  filter(!is.na(Genus)) %>%
  group_by(BRD_affected_or_healthy_s, Genus) %>%
  tally() %>%
  spread(key = BRD_affected_or_healthy_s,
         value = n) %>%
  arrange(desc(BRD), desc(healthy)) %>%
  head(5) %>%
  kable()
  
```



In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
